patches/0003-Avoid-using-_ZNSsC1ERKSsjRKSaIcE-GLIBCXX-symbol-due-.patch

```diff
@@ -1,28 +0,0 @@
-From 21dfe5a22da02ed8bcd9fa1cca7c18c8c5310172 Mon Sep 17 00:00:00 2001
-From: patches <patches@localhost>
-Date: Sun, 28 Apr 2019 18:45:57 +0000
-Subject: [PATCH 3/4] Avoid using _ZNSsC1ERKSsjRKSaIcE GLIBCXX symbol due to
- version compat issues
-
----
- src/common/dwarf/elf_reader.cc | 4 ++--
- 1 file changed, 2 insertions(+), 2 deletions(-)
-
-diff --git a/src/common/dwarf/elf_reader.cc b/src/common/dwarf/elf_reader.cc
-index 4135a51a..bfaaeefe 100644
---- a/src/common/dwarf/elf_reader.cc
-+++ b/src/common/dwarf/elf_reader.cc
-@@ -1202,8 +1202,8 @@ const char *ElfReader::GetSectionInfoByName(const string &section_name,
- 
- bool ElfReader::SectionNamesMatch(const string &name, const string &sh_name) {
-   if ((name.find(".debug_", 0) == 0) && (sh_name.find(".zdebug_", 0) == 0)) {
--    const string name_suffix(name, strlen(".debug_"));
--    const string sh_name_suffix(sh_name, strlen(".zdebug_"));
-+    const string name_suffix(&name.c_str()[strlen(".debug_")]);
-+    const string sh_name_suffix(&sh_name.c_str()[strlen(".zdebug_")]);
```

Comment:
Surprised this isn't still needed - without it breakpad couldn't load with the libstdc++ included with several games.

Reply:
Took a bit of digging but I believe I deleted that patch because the matching code section got rewritten in breakpad.

https://github.com/google/breakpad/blob/255dbbd061a400e7a3f601e82a62e65058b39e5e/src/common/dwarf/elf_reader.cc#L1230-L1233

At the very least this shouldn't be an instance of
`std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::string const&, unsigned int, std::allocator<char> const&)` anymore.